javascript/builtins/intl/NumberFormat.json

```diff
@@ -571,40 +571,88 @@
             },
             "options_useGrouping_parameter": {
               "__compat": {
-                "description": "<code>options.useGrouping</code> parameter",
```

Comment:
This should be added back!

Reply:
Done. Sorry - no idea why I would have removed that.